src/Avatar/Avatar.tsx

```diff
@@ -81,14 +85,18 @@ export type AvatarBaseProps = {
 
   /** Custom ImageComponent for Avatar. */
   ImageComponent?: React.ComponentClass;
-};
+} & InlinePressableProps;
 
 /** Avatars are found all over ui design from lists to profile screens.
  * They are commonly used to represent a user and can contain photos, icons, or even text. */
 export const AvatarBase: RneFunctionComponent<AvatarBaseProps> = ({
```

Comment:
You forgot to add `onPressIn` and `onPressOut` to `AvatarBaseProps`

Reply:
`InlinePressableProps` this includes onPressIn and onPressOut